Code/src/main/java/nl/utwente/group10/ui/components/lines/ConnectionLine.java

```diff
@@ -47,15 +50,21 @@ public void setStartPosition(double xCoord, double yCoord) {
         updateBezierControlPoints();
     }
 
+    /**
+     * Sets the start coordinates for this ConnectionLine object.
+     * 
+     * @param point
+     *            X and Y coordinate local to parent.
+     */
     public void setStartPosition(Point2D point) {
         setStartPosition(point.getX(), point.getY());
     }
 
     /**
-     * Sets the end coordinate for this Line object.
+     * Sets the end coordinates for this ConnectionLine object.
      *
-     * @param xCoord
-     * @param yCoord
+     * @param xCoord X coordinate local to parent.
```

Comment:
Yes, usually, but nothing in this class is setting its parent to the CustomUIPane, so i didnt mention that.